Controls/ConnectionControl.cs

```diff
@@ -150,6 +150,12 @@ private void CMB_sysid_SelectedIndexChanged(object sender, EventArgs e)
                     if (MainV2.comPort.MAV.param.Count == 0 && !(Control.ModifierKeys == Keys.Control))
                         MainV2.comPort.getParamList();
 
+                    //enable the payload control page if a mavlink gimbal is detected

```

Comment:
please start using the mavchanged event, vs placing code everywhere to do this.

Reply:
thanks, changed to use the mavchanged event